# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pdfedit/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pdfedit-0.3.2/src/utils/confparser.h.vanilla	2008-05-14 14:50:26.000000000 +0200
+++ pdfedit-0.3.2/src/utils/confparser.h	2008-05-14 14:50:44.000000000 +0200
@@ -66,6 +66,7 @@
 #ifndef _CONFPARSER_H_
 #define _CONFPARSER_H_
 
+#include<string.h> // mem*, str*
 #include<iostream>
 #include<string>
 #include<vector>
